designs/Compatibility.md

```diff
@@ -16,6 +16,15 @@ The built-in system treats `CharSet.None` as `CharSet.Ansi`. The P/Invoke source
 
 [`CallingConvention`](https://docs.microsoft.com/dotnet/api/system.runtime.interopservices.dllimportattribute.callingconvention) will not be supported for `GeneratedDllImportAttribute`. Users will be required to use the new `UnmanagedCallConvAttribute` attribute instead. This attribute provides support for extensible calling conventions and provides parity with the `UnmanagedCallersOnlyAttribute` attribute and C# function pointer syntax. We will enable our conversion code-fix to automatically convert explicit and known calling convention usage to use the `UnmanagedCallConvAttribute`.
 
+### Required references
+
+The following framework references are required:
+- `System.Memory`
+- `System.Runtime`
+- `System.Runtime.InteropServices`
+
+These are all part of `NETCoreApp` and will be referenced by default unless [implicit framework references are disabled](https://docs.microsoft.com/dotnet/core/project-sdk/msbuild-props#disableimplicitframeworkreferences).
```

Comment:
Can you go through this and confirm all instance of "NetCoreApp" have the same casing? I see above we use "Net" and not "NET". I've no preference.

Reply:
```suggestion
These are all part of `NetCoreApp` and will be referenced by default unless [implicit framework references are disabled](https://docs.microsoft.com/dotnet/core/project-sdk/msbuild-props#disableimplicitframeworkreferences).
```